armeria-backend/src/main/scala/sttp/client3/armeria/AbstractArmeriaBackend.scala

```diff
@@ -59,7 +59,7 @@ abstract class AbstractArmeriaBackend[F[_], S <: Streams[S]](
   override def responseMonad: MonadError[F] = monad
 
   override def send[T, R >: SE](request: Request[T, R]): F[Response[T]] =
-    adjustExceptions(request)(execute(request))
+    monad.unit(request).flatMap(r => adjustExceptions(request)(execute(r)))
```

Comment:
ah, good catch! though for this purpose there's `monad.suspend`

Reply:
🎖️